README.md

```diff
@@ -1,16 +1,56 @@
 # [McHacks 7](https://2020.mchacks.ca)
 
-This repository contains the code behind the static site of McHacks 7.
-
 [![Netlify Status](https://api.netlify.com/api/v1/badges/29fc0619-7608-47de-8105-727d647ba4aa/deploy-status)](https://app.netlify.com/sites/mchacks7/deploys)
 
+This repository contains the code behind the static site of McHacks 7.
+
 ## Setup
 
 1. Make sure you have [node](https://nodejs.org/en/) and [yarn](https://yarnpkg.com/lang/en/) installed.
 2. Run `yarn global add gatsby-cli` to install Gatsby CLI locally.
 3. Run `yarn` to install dependencies.
 4. Run `gatsby develop` to start dev server! 🚀
 
+## Scripts
+
+**Start the development server:**
+
+`yarn start` or `gatsby develop`
+
+**Build the website:**
+
+`yarn build` or `gatsby build`
+
+**Start the production server:**
+
+`yarn serve` or `gatsby serve`
+
+**Format code:**
+
+`yarn format`
+
+## Folder Structure
+
+    ├── docs                    # Documentation files
+    ├── src                     # Source files
+    │   ├── components          # Page sections files
+    │   ├── images              # Image files
+    │   └── pages               # Page files
+    │   └── styles              # Style files
+    │   │   └── components
+    │   │   └── font
+    ├── static                  # Unbundled assets
+
 ## Contributing
 
-See our [contributing guide](https://github.com/hackmcgill/mchacks7/blob/develop/CONTRIBUTING.md).
+> Want to contribute to the McHacks site?
+
+See our [contributing guide](https://github.com/hackmcgill/mchacks7/blob/develop/docs/CONTRIBUTING.md).
+
+## Deployment
+
+We are using Netlify to compile and host our code. When a PR is created, Netlify builds the site and generates a deploy preview to confirm everything is working as expected. Once code is merged to `master` branch, Netlify will promote the code to production at `2020.mchacks.ca`. Netlify also handles the SSL certificate for this site.
```

Comment:
Nit: the SSL for [mchacks7.netlify.com](mchacks7.netlify.com) is handled by Netlify, but the SSL for [2020.mchacks.ca](2020.mchacks.ca) and [mchacks.ca](mchacks.ca) (and their .io equivalents) is handled by Cloudflare.